ParisBNF/abb/BNFabb105.xml

```diff
@@ -0,0 +1,979 @@
+                             <hi rend="rubric">ቅድስት፡ ወብፅዕ</hi>ት᎓ ስብሕት᎓ ወቡርክት፨ ክብርት᎓ ወልዕልት፨ አንቀጸ᎓ ብርሃን᎓
+                             መዐርገ᎓ ሕይወት᎓</incipit>
+
+                             <msItem xml:id="ms_i1.6.1">
+                               <locus from="181v" to="18vb"/>
+                               <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ ዱያን፡</hi></title>
+                               <textLang mainLang="gez"></textLang>
+                                 <incipit xml:lang="gez">ወከዕበ፡ ናስተበቍዖ፡ ለእግዚአ፡ ኵሉ፡ አምላክነ፡ አኃዜ፡ ኵሉ፡ ዘይነግሥ፡ ለኵሉ፡ ዓለም፡
+                                   አቡሁ፡ ለእግዚእነ፡ ወመድኀኒነ፡ ኢየሱስ፡ ክርስቶስ፡ ኀዋጺሃ፡ ለነፍስ፡ ወመፈውሰ፡ ሥጋ፡
+                               </incipit>
+                             </msItem>
+
+                             <msItem xml:id="ms_i1.6.2">
+                               <locus from="181vb" to="182rb"/>
+                               <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ እለ፡ ይነግድ፡</hi></title>
+                               <textLang mainLang="gez"></textLang>
+                                 <incipit xml:lang="gez">ወከዕበ፡ ናስተበቍዖ፡ ለእግዚአ፡ ኵሉ፡ አምላክነ፡ አኃዜ፡ ኵሉ፡ ዘይነግሥ፡ ለኵሉ፡ ዓለም፡
+                                   አቡሁ፡ ለእግዚእነ፡ ወመድኀኒነ፡ <pb n="182ra"/>ኢየሱስ፡ ክርስቶስ፡ ዘይሁብ፡
+                                    ፍኖተ፡ ባሕር፡ ዘይሁብ፡ ፍኖተ፡ ባሕር፡ ወስተ፡ ባሕር፡
+                               </incipit>
+                             </msItem>
+
+                             <msItem xml:id="ms_i1.6.3">
+                               <locus from="182rb" to="182vb"/>
+                               <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ ዝናማት፡</hi></title>
+                               <textLang mainLang="gez"></textLang>
+                                 <incipit xml:lang="gez">ወከዕበ፡ ናስተበቍዖ፡ ለእግዚአ፡ ኵሉ፡ አምላክነ፡ አኃዜ፡ ኵሉ፡ ዘይነግሥ፡ ለኵሉ፡ ዓለም፡
+                                   አቡሁ፡ ለእግዚእነ፡ ወመድኀኒነ፡ ኢየሱስ፡ ክርስቶስ፡ እግዚኦሙ፡ ወአምላኮሙ፡ ለማያት፡ ንስእለከ፡ ወናስተበቍዐከ፡ ከመ፡ ትፈኑ፡ ዝናመ፡
+                               </incipit>
+                             </msItem>
+                             <msItem xml:id="ms_i1.6.4">
+                               <locus from="182vb" to="183ra"/>
+                               <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ ዝናማት፡</hi></title>
+                               <textLang mainLang="gez"></textLang>
+                                 <incipit xml:lang="gez">ወከዕበ፡ ናስተበቍዖ፡ ለእግዚአ፡ ኵሉ፡ አምላክነ፡ አኃዜ፡ ኵሉ፡ ዘይነግሥ፡ ለኵሉ፡ ዓለም፡
+                                   አቡሁ፡ ለእግዚእነ፡ ወመድኀኒነ፡ ኢየሱስ፡ ክርስቶስ፡ እግዚኦሙ፡ ወአምላኮሙ፡ ለማያት፡ ንስእለከ፡ ወናስተበቍዐከ፡ ከመ፡ ትፈኑ፡ ዝናመ፡
+                               </incipit>
+                             </msItem>
+
+                             <msItem xml:id="ms_i1.6.5">
+                               <locus from="183ra" to="183rb"/>
+                               <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ ማይተ፡ አፍላግ፡</hi></title>
```

Comment:
ref="LIT4711Mastabqwe#SupplicationRivers"

Reply:
thanks

Reply:
fols 183 - 184 ጸሎት፡ በእንተ፡ እለ፡ ያበውኡ፡
መባአ፡ is Supplication for those who bring gifts, not in 4711. Mayby old, but maybe from the series of supplication as in Sinodos. SHould be nevertheless indroduced as # under 4711?